armsrc/appmain.c

```diff
@@ -440,15 +434,14 @@ void printStandAloneModes(void) {
 	DbpString("   LF HID corporate 1000 bruteforce - (Federico dotta & Maurizio Agazzini)");
 #endif 
 #if defined(WITH_HF_MATTYRUN)
-	DbpString("   HF Mifare sniff/clone - aka MattyRun (Mat�as A. R� Medina)");
+	DbpString("   HF Mifare sniff/clone - aka MattyRun (Mat�as A. R� Medina)");
 #endif 
 #if defined(WITH_HF_COLIN)
     DbpString("   HF Mifare ultra fast sniff/sim/clone - aka VIGIKPWN (Colin Brigato)");
 #endif
 	
 	//DbpString("Running ");	
 	//Dbprintf("  Is Device attached to USB| %s", USB_ATTACHED() ? "Yes" : "No"); 
-	//Dbprintf("  Is Device attached to FPC| %s", 0 ? "Yes" : "No"); 
```

Comment:
dont remove my work-in-progress code

Reply:
Merge conflict with old version from iceman fork.